internal/utils/codecrafters_client.go

```diff
@@ -190,6 +199,9 @@ func (c CodecraftersClient) FetchBuild(buildId string) (FetchBuildStatusResponse
 			}
 
 			if fetchBuildResponse.Status != "failure" && fetchBuildResponse.Status != "success" {
+				customError := &CustomError{Message: "unexpected submission status: " + fetchBuildResponse.Status}
+				sentry.CaptureException(customError)
+
```

Comment:
We probably need monitoring here too btw: https://github.com/codecrafters-io/test-runner/blob/44ea70a6e03cc72f4a346a7f38636f4ccb2c2472/handle_git_push_command.go#L147